src/core/thread/mle.cpp

```diff
@@ -1733,6 +1739,10 @@ ThreadError Mle::HandleDataResponse(const Message &aMessage, const Ip6::MessageI
             mNetif.GetActiveDataset().Set(activeTimestamp, aMessage, offset + sizeof(tlv), tlv.GetLength());
         }
     }
+    else
+    {
+        VerifyOrExit(update, ;);
```

Comment:
Will update here.
5.15.3 mentions optionally an MLE Pending Timestamp TLV that matches. So I only compare Active Timestamp here.